src/LatentGPs.jl

```diff
@@ -1,5 +1,26 @@
 module LatentGPs
 
-greet() = print("Hello World!")
+using Distributions
+using KernelFunctions
+using AbstractGPs
+using LinearAlgebra
+using Random
+using Functors
+
+import Statistics
+import Distributions
+
+export LatentGP
+
+export logpdf, rand, trainable, randf, randy
```

Comment:
It seems currently only
```suggestion
export logpdf, rand
```
are defined.